about.md

```diff
@@ -34,3 +34,10 @@ Check `.github/actions/aggregate-members-location/action.yml` to see how it's do
 
 [Add me to the map]:https://github.com/{{ site.github_project }}/actions/workflows/add_me_to_the_map.yml
 [Remove me from the map]:https://github.com/{{ site.github_project }}/actions/workflows/remove_me_from_the_map.yml
```

Comment:
What about the existing content above?  I think some users will not want to keep it.  I though that we would probably not update ofter the `about.md` page, and that uses could directly change the text here.  If we add a system to add about fragments, maybe the above instructions should be moved in `_about/10_faq.md` or a bunch of `_about/10_who_can_appear_on_this_map.md`, etc. ?